packages/node/src/utils/project.ts

```diff
@@ -42,7 +42,8 @@ export async function getChainTypes(
 ): Promise<ChainTypes> {
   // If the project is load from local, we will direct load them
   if (reader instanceof LocalReader) {
-    return loadChainTypes(file, root);
+    const raw = loadChainTypes(file, root);
+    return parseChainTypes(raw);
```

Comment:
It seems every path for returning does `return parseChainTypes`. Reworking this function so there is a single return would reduce chances of the issue happening again